packages/web/src/scss/components/Offcanvas/README.md

```diff
@@ -0,0 +1,99 @@
+# Offcanvas
+
+Offcanvas is a sidebar component that can be toggled via JavaScript to appear from the right edge of the viewport.
+Buttons or anchors are used as triggers that are attached to specific elements you toggle, and data attributes are used to invoke JavaScript plugin.
+
+## JavaScript Plugin
+
+For full functionality, you need to [provide Spirit JavaScript][js-plugin-readme], which will handle
+toggling of the Header component:
```

Comment:
Header? 

It is first time mentioned :)

Reply:
Fixed.